src/v3/test/integration/__snapshots__/authenticator-enroll-data-phone.test.tsx.snap

```diff
@@ -5,6 +5,7 @@ exports[`authenticator-enroll-data-phone should render form 1`] = `
   <span>
     <main
       class="ods-4o5zYQ ods-76eppy ods-66Tc65 ods-6lar65 ods-2oc5SK ods-4XAbls ods-2NLKpK ods-1ll9ky ods-4IdjLs auth-container main-container mainViewContainer"
+      id="generated"
```

Comment:
remove mask function in test

Reply:
I will do a find and delete on that updateDynamicAttr function in the integration tests.